app/views/jobs/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_for([@company, @job]) do |f| %>
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+
+  <%= f.label :level_of_interest %>
+  <%= f.number_field :level_of_interest %>
+
+  <%= f.collection_select(:category_id, Category.all, :id, :title, { prompt: "Select Category" }) %>
```

Comment:
here is a good example of too much logic in the view. you could set `Category.all` to a variable and send it in!